style.less

```diff
@@ -121,7 +126,80 @@ button {
             padding: 10px 20px;
             border: none;
             font-size: 150%;
-            color: white;
+            color: @white;
         }
     }
+}
+
+@back-button-size: 40px;
+.back-button {
+    left: 10px;
+    top: 25px;
+    width: @back-button-size;
+    height: @back-button-size;
+    line-height: @back-button-size;
+    position: absolute;
+    border: none;
+    background: none;
+    font-size: 200%;
+    padding: 0;
+    color: @black;
+    &.white {
+        color: @white;
+    }
+}
+
+//Scroller: http://codepen.io/shakdaniel/pen/JoKOQx
```

Comment:
We're actually not using this at the moment, I couldn't easily and quickly fit it in. Maybe we'll get it in later :)